drape/overlay_handle.cpp

```diff
@@ -232,18 +232,17 @@ std::string SquareHandle::GetOverlayDebugInfo()
 /// @param[in] depth        Manual priority from styles (bigger is better)
 uint64_t CalculateOverlayPriority(int minZoomLevel, uint8_t rank, float depth)
 {
+  ASSERT_GREATER_OR_EQUAL(depth, 0, ());
+  ASSERT_LESS_OR_EQUAL(depth, 100000, ());
```

Comment:
nit: I'd prefer one expression here like:
ASSERT(0 <= depth && depth <= 100000, (depth));

Reply:
Why 100000 is used? Can we name this constant or explain it?

Reply:
No idea why 100000 was used here. Probably an arbitrary number to cap the priorities "just in case".
Actually I see no point in capping overlay priorities from above. The only real limits I see are imposed by the unsigned int32 (because int32 is used for passing priorities via protobuf).